components/molecules/TeamMemberRow/team-member-row.tsx

```diff
@@ -7,39 +7,72 @@ import { TeamMemberData } from "../TeamMembersConfig/team-members-config";
 
 interface TeamMemberRowProps extends TeamMemberData {
   className?: string;
+  onDelete: Function;
+  onUpdate: Function;
```

Comment:
From usage I understood that `onDelete` is a function that takes 1 argument, and `onUpdate` is a function that takes 2 arguments. This should be clearly defined in the type and have a strict structure required from the calling function. `Function` is too generic.